### PR TITLE
Improve samlsso response ui

### PR DESCRIPTION
**Pull request by kanchanagodage**
Improved samlsso response ui
Capture

Resolve https://github.com/wso2/product-is/issues/9395

### CODE REVIEW

File: features/org.wso2.carbon.identity.sso.saml.server.feature/resources/samlsso_response.html [hunk 1]
```diff
@@ -1,20 +1,104 @@
+<!DOCTYPE html>
+
+<!--
+~    Copyright (c) 2020, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```
Comment (jeradrutnam) [inline features/org.wso2.carbon.identity.sso.saml.server.feature/resources/samlsso_response.html:4]:
```suggestion
~    Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```

Comment (jeradrutnam) [inline features/org.wso2.carbon.identity.sso.saml.server.feature/resources/samlsso_response.html]:
```suggestion
~    Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```

**Review by jeradrutnam — APPROVED** [review]

File: features/org.wso2.carbon.identity.sso.saml.server.feature/resources/samlsso_response.html [hunk 2]
```diff
@@ -1,20 +1,104 @@
+<!DOCTYPE html>
+
+<!--
+~    Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```
Comment (omindu) [inline features/org.wso2.carbon.identity.sso.saml.server.feature/resources/samlsso_response.html:4]: Shouldn't this be ASL 2.0 ?

File: features/org.wso2.carbon.identity.sso.saml.server.feature/resources/samlsso_response.html [hunk 3]
```diff
@@ -1,20 +1,104 @@
...
-        	If the redirection fails, please click the post button.</p>
 
-        	<form method='post' action='$acUrl'>
-       			<p>
-					<!--$params-->
+<head>
+    <title>Redirecting</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (madurangasiriwardena) [inline features/org.wso2.carbon.identity.sso.saml.server.feature/resources/samlsso_response.html:22]: why do we rely on a CDN for this stylesheet?

Comment (kanchanagodage) [reply features/org.wso2.carbon.identity.sso.saml.server.feature/resources/samlsso_response.html:22]: This is a single HTML file, we're not able to import external files

**Pull request merged**